proto-public/pbmesh/v1alpha1/computed_routes.proto

```diff
@@ -112,7 +117,18 @@ message BackendTargetDetails {
   // identity info
   BackendReference backend_ref = 1;
 
-  hashicorp.consul.catalog.v1alpha1.Service service = 3;
-  hashicorp.consul.catalog.v1alpha1.FailoverPolicy failover_policy = 4;
-  DestinationPolicy destination_policy = 5;
+  hashicorp.consul.catalog.v1alpha1.FailoverConfig failover_config = 2;
+  DestinationConfig destination_config = 3;
+
+  // ServiceEndpointsID is not populated naturally and the field exists only for
+  // downstream consumers.
+  hashicorp.consul.resource.ID service_endpoints_id = 4;
+
+  // ServiceEndpoints is not populated naturally and the field exists only for
+  // downstream consumers.
+  hashicorp.consul.catalog.v1alpha1.ServiceEndpoints service_endpoints = 5;
+
+  // IdentityRefs are not populated naturally and the field exists only for
+  // downstream consumers.
+  repeated hashicorp.consul.resource.Reference identity_refs = 6;
```

Comment:
These are validated at admission time to force them to be `nil`. These are here so that during the sidecar controller resource assembly we can park data for the ultimate backend targets next to the rest of the data for them without introducing a second parallel map.